module/modmanager/manager.go

```diff
@@ -87,15 +92,17 @@ type addedResource struct {
 
 // Manager is the root structure for the module system.
 type Manager struct {
-	mu                      sync.RWMutex
-	logger                  logging.Logger
-	modules                 map[string]*module
-	parentAddr              string
-	rMap                    map[resource.Name]*module
-	untrustedEnv            bool
-	removeOrphanedResources func(ctx context.Context, rNames []resource.Name)
-	restartCtx              context.Context
-	restartCtxCancel        context.CancelFunc
+	mu                        sync.RWMutex
+	logger                    logging.Logger
+	modules                   map[string]*module
+	parentAddr                string
+	rMap                      map[resource.Name]*module
+	untrustedEnv              bool
+	viamHomeDir               string
```

Comment:
Can we document here the expected values this (and `modulateDataParentDirectory`) can take on a post-initialized `Manager` object? When I read variables like these in code I'm always wondering what is `openFile(str)` going to find?
* Can the string contain env variables (e.g: `$HOME`) or other things that the program will evaluate/substitute before passing to the OS?
* Can the string be a relative path? Is it relative to the process PWD (os evaluated)? Is it relative to some other directory-representing string (program evaluated)?
* Is the string guaranteed to be an absolute path?